### PR TITLE
Add blog section to homepage

### PR DESCRIPTION
**Pull request by broskoski**
screen shot 2018-06-26 at 1 11 53 pm

### CODE REVIEW

File: lib/truncate.coffee [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{ take } = require 'underscore'
+
+module.exports = (text, length = 50) ->
+  return unless text
+  tokens = text.split ' '
+  truncated = take(tokens, length).join(' ')
+  truncated = truncated + "..." if tokens.length > length
+  truncated
```
Comment (dzucconi) [inline lib/truncate.coffee]: `{ truncate } = require 'underscore.string'`

Comment (broskoski) [reply lib/truncate.coffee]: Our truncate ensures that words don't get cut off, underscore.string does it by the character.

Comment (dzucconi) [reply lib/truncate.coffee]: works for me

Comment (dzucconi) [reply lib/truncate.coffee]: but i still dunno why you writing coffeescript these days

Comment (broskoski) [reply lib/truncate.coffee]: Cause I'm a maniac

Comment (broskoski) [reply lib/truncate.coffee]: This was actually already written, I just put it in its own file.

Comment (dzucconi) [reply lib/truncate.coffee]: tru

**Pull request merged**